qualang_tools/bakery/readme.md

```diff
@@ -0,0 +1,273 @@
+# Waveform baking
+This library introduces a new framework for creating arbitrary waveforms and storing them in a usual configuration file. 
+The idea is to simplify the writing of pulse samples that would usually be declared as incompatible with respect to QUA restrictions for uploading the samples to the OPX. A nice advantage of using this tool is to embed into one single waveform a series of instructions that allows program memory preservation.

```

Comment:
The idea is to simplify the writing of pulse samples **without the limitation imposed by the hardware**. Using this tool provides an advantage by embedding into one single waveform a series of instructions that allows program memory preservation.

Note - The limitation is not from QUA, because QUA is (supposed/planned) to be hardware neutral.